src/subcommands/api.ts

```diff
@@ -0,0 +1,74 @@
+import { Args } from "../args.ts";
+import { API } from "../utils/mod.ts";
+import TokenProvisioner from "../utils/access_token.ts";
+import { error } from "../error.ts";
+import { wait } from "../utils/spinner.ts";
+
+const help = `Perform API calls to any endpoint of the Deploy API (ALPHA)
+
+EXAMPLES:
+
+Get the details of an organization:
+
+    deployctl api organizations/04f19625-35d3-4c05-857e-bcaa3b0af374
+
+Create a project in an organization:
+
+    deployctl --method=POST --body='{"name": "my-project"}' organizations/04f19625-35d3-4c05-857e-bcaa3b0af374/projects
+
+You can find the specification of the API in https://apidocs.deno.com
+
+USAGE:
+    deployctl api [OPTIONS] <ENDPOINT>
+
+OPTIONS:
+    -h, --help                 Prints this help information
+        --method=<HTTP-METHOD> HTTP method to use (defaults to GET)
+        --body=<JSON>          Body of the request. The provided string is sent as is to the API
+        --token=<TOKEN>        The API token to use (defaults to auto-provisioned token)
+`;
+
+export default async function (args: Args): Promise<void> {
+  if (args.help) {
+    console.log(help);
+    Deno.exit(0);
+  }
+  let endpoint = args._.shift()?.toString();
+  if (!endpoint) {
+    error(
+      "Missing endpoint positional argument. USAGE: deployctl api <endpoint>",
+    );
+  }
+  if (!endpoint.startsWith("/")) {
+    endpoint = `/${endpoint}`;
+  }
+  if (!args["api-version"]) {
+    endpoint = `/v1${endpoint}`;
+  } else if (args["api-version"] !== "0") {
+    endpoint = `/${args["api-version"]}${endpoint}`;
+  }
```

Comment:
What kind of values are expected to come as a value of `api-version`? For instance if a user provides `--api-version=1` (from what they might guess based on `!== "0"`), the endpoint will be mutated to `/1/foo/bar` but it wouldn't work - so maybe doing a simple validation against the value of `api-version` would be beneficial?

Reply:
I think I prefer to leave it as open as possible. As said before, this is option is not something the regular user should concern about at all, it's only an internal diagnostics option. The user doesn't need to do `--api-version=1` or any other version, because the subcommand already works by default against the only public API version available, v1. 
Leaving it open to any version can be handy in the future when there's a new experimental API version, for example.

Reply:
That's a good point, let's keep it as is then.
One more question though: what about something like this? This would require specifying `--api-version=v0` (instead of `--api-version=0`) when they want to call v0 endpoints, but the code would be a little simpler and it's intuitive that the value specified in `--api-version` is set as a prefix of the request path without any modification 

```suggestion
  if (!args["api-version"]) {
    endpoint = `/v1${endpoint}`;
  } else {
    endpoint = `/${args["api-version"]}${endpoint}`;
  }
```

Reply:
Oh, I missed the fact that there's a `v0` alias. I'll do that yes

Reply:
@magurotuna Now I'm thinking that maybe this whole `--api-version` option was a bad idea. I'm leaning now towards just doing
```javascript 
if (!/^\/v\d+\//.test(endpoint)) {
  endpoint = `/v1${endpoint}`;
}
```

This would let users that want to use a version other than v1, to just prefix the endpoints with such version (ie `api /v0/user`)

Thoughts?

Reply:
Yeah that sounds reasonable and much better 👍